proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
```

Comment:
How?

Reply:
MPWG would like to have a no-cost compatibility logo. That means any implementation that passes the TCK could use the compatibility logo. To your point in the MicroProfile google group, we have not yet discussed budget. We do need to discuss  funding MPWG. However, I don't think the CN4J can add cloud-related projects without increasing the fee structure.

Reply:
That's not in the proposal.